tests/Paket.Tests/ProjectFile/InterProjectDependencySpecs.fs

```diff
@@ -37,8 +37,8 @@ let ``should detect relative path for dependencies in Project2 proj file``() =
         ProjectFile.TryLoad("./ProjectFile/TestData/Project2.fsprojtest").Value.GetInterProjectDependencies()
         |> List.map (fun p -> p.RelativePath)
 
-    paths.[0] |> shouldEqual "..\\..\\src\\Paket\\Paket.fsproj"
-    paths.[1] |> shouldEqual "..\\Paket.Core\\Paket.Core.fsproj"
+    paths.[0] |> shouldEqual "..\\..\\..\\..\\src\\Paket\\Paket.fsproj"
+    paths.[1] |> shouldEqual "..\\..\\..\\..\\src\\Paket.Core\\Paket.Core.fsproj"
```

Comment:
The unit tests used it.  From where it is called by nunit, those projects do not exist in the paths that were specified.